languages/translations/en.json

```diff
@@ -50,6 +50,11 @@
                 "destination": "Take the ferry towards {destination}"
             }
         },
+        "phrase": {
+            "two linked by distance": "{instruction_one} then in {distance} {instruction_two}",
```

Comment:
I think the spaces should be removed here.

Reply:
I don't see any camel casing in here, so I've merged this like this.

`two linked by distance` => `twolinkedbydistance`

Reply:
@mcwhittemore retracting my comment, this should spaces.